hoot-core/src/main/cpp/hoot/core/visitors/RemoveDuplicateWayNodesVisitor.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2018 DigitalGlobe (http://www.digitalglobe.com/)
+ */
+
+#ifndef REMOVEDUPLICATEWAYNODESVISITOR_H
+#define REMOVEDUPLICATEWAYNODESVISITOR_H
+
+// Hoot
+#include "ElementOsmMapVisitor.h"
+
+namespace hoot
+{
+
+/**
+ * Removes all any way nodes that are duplicates, with the exception of start/end node duplicates.
```

Comment:
Should be `Removes all way nodes` or `Removes any way nodes` but not `Removes all any way nodes`

Reply:
Ha.  words are hard.  Yeah, I'll commit a fix...thanks for looking at this.